src/index.html

```diff
@@ -11,7 +11,175 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field">
+          <span>Surname:</span>
+          <input
+            type="text"
+            name="surname"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label  class="field">
+          <span>Name:</span>
+          <input
+            type="text"
+            name="name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="field">
+          <span>How old are You?</span>
+          <input
+          type="number"
+          name="Age"
+          value="12"
+          min="1"
+          max="100"
+          >
```

Comment:
please be consistent in code review - you're changing only the parts explicitly mentioned, but don't apply same logic for the rest of the code
indentation is not fixed

please fix this across entire code
```suggestion
          <input
            type="number"
            name="Age"
            value="12"
            min="1"
            max="100"
          >
```

Reply:
> please be consistent in code review - you're changing only the parts explicitly mentioned, but don't apply same logic for the rest of the code indentation is not fixed
> 
> please fix this across entire code - review EACH input and make sure that attributes are indented

Hello, I believe I checked for ALL, I don't understand why it is not reflected!

Reply:
I just checked my code, these parts that you mentioned (lines 44-48) has correct indentation in VScode! maybe I haven't commited the changes in the right way!